### ISSUE TITLE
ModuleNotFoundError: No module named 'chardet'

### ISSUE THREAD

**Issue by RvV1979** [moiexpositoalonsolab/grenepipe#37]
Hi, 
FYI after (re-)installing grenepipe I received the error below:
```
Creating conda environment /grenepipe-0.12.2/rules/../envs/bcftools.yaml...
Downloading and installing remote packages.
CreateCondaEnvironmentException:
Could not create conda environment from /grenepipe-0.12.2/rules/../envs/bcftools.yaml:
Traceback (most recent call last):
  File "grenepipe/lib/python3.7/site-packages/requests/compat.py", line 11, in <module>
    import chardet
ModuleNotFoundError: No module named 'chardet'
```
This was resolved after running `conda install -c anaconda chardet` from within the grenepipe environment. This may be an issue specific to my system (in which case it can be ignored) but else it may be something to add in the install scripts.

**Comment by lczech** [comment]
Hey @RvV1979,

thanks for reporting this! I'll see if I can just add this to the env file, I think that should work. Let's keep this open, and I'll have a look at it soon!

Cheers
Lucas

**Comment by lczech** [comment]
Okay, fixed in c48c453ed5572307c1a3519a93905b094e8108d6, thanks for reporting!
